crates/turbopack-build/Cargo.toml

```diff
@@ -22,6 +22,8 @@ indoc = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 serde_qs = { workspace = true }
+sourcemap = "6.0.2"
+swc_core = { workspace = true, features = ["__parser", "ecma_minifier"] }
```

Comment:
Please add `ecma_minifier_concurrent` to the list of features.

https://github.com/swc-project/swc/blob/9daa0e2c6a6d31e8694d180408362d5f6883403e/crates/swc_core/Cargo.toml#L48-L49